Composer/packages/client/src/pages/home/Home.tsx

```diff
@@ -234,6 +240,16 @@ const Home: React.FC<RouteComponentProps> = () => {
           <WhatsNewsList newsList={feed.whatsNewLinks} />
         </div>
       </div>
+      {warnAboutDotNet && (
+        <InstallDepModal
+          downloadLink={missingDotnetVersionError.link.url}
+          downloadLinkText={formatMessage('Install .NET Core SDK')}
```

Comment:
Nit: It seems strange that the learn more link has text and URL, but the download link has URL and hard-coded text.